modules/standard/BigInteger.chpl

```diff
@@ -4342,11 +4333,49 @@ module BigInteger {
                     ``zero`` if unspecified
      :type rounding: ``round``
    */
-  @deprecated(notes="bigint.divR2Exp method is deprecated - please use the standalone function :proc:`~BigInteger.divR2Exp`")
+  @deprecated(notes="bigint.divR2Exp method is deprecated - please use the standalone function :proc:`~BigInteger.rem2Exp` with :enum:`roundingMode` instead")
   proc bigint.divR2Exp(const ref numer: bigint,
                                  exp: integral,
-                       param     rounding = round.zero) {
-    BigInteger.divR2Exp(this, numer, exp, rounding);
+                       param     rounding = round.zero)
+    do BigInteger.rem2Exp(this, numer, exp, chpl_roundToRoundingMode(rounding));
+
+
```

Comment:
I think we've got an extra new line here, might want to remove that and get all functions to 1 new line apart.